### PR TITLE
Pipelined opengl

### PR DESCRIPTION
**Pull request by mads256h**
We are now working with a pipeline
That is each stage renders to a framebuffer (or gives some other output) This framebuffer (or other output) is then inputted into the next stage that then again renders to a framebuffer or other output.

We have a special stage the `DrawFramebufferStage`. It takes a framebuffer and outputs it on the screen. Thus it has no output.

### CODE REVIEW

**Comment by mads256h** [comment]
We still need to provide actual screen resolution and when it updates

File: app/src/main/java/dk/scuffed/opengltest/OpenGLView.kt [hunk 1]
```diff
@@ -5,12 +5,12 @@ import android.opengl.GLSurfaceView
 
 class OpenGLView(context: Context) : GLSurfaceView(context) {
 
-    private val renderer: OpenGLRenderer
+    //private val renderer: OpenGLRenderer
```
Comment (S3JER) [inline app/src/main/java/dk/scuffed/opengltest/OpenGLView.kt]: delete plz

**Review by S3JER — APPROVED** [review]
LGTM

**Review by Casper-NS — APPROVED** [review]
lgtm

**Pull request merged**